mock-vm/types/src/address.rs

```diff
@@ -11,7 +11,7 @@ pub const CONTRACT_ADDRESS_PREFIX: u32 = 0x0000cadd;
 /// Blockchain-agnostic address representation.
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, BorshSerialize, BorshDeserialize)]
 pub struct Address {
-    data: [u8; ADDRESS_LENGTH]
+    pub data: [u8; ADDRESS_LENGTH]
```

Comment:
I'd not expose it. Add `inner_bytes(&self) -> &[u8];`.

Reply:
Ok